app/ui.js

```diff
@@ -1861,13 +1860,15 @@ const UI = {
     // Apply remote resizing or local scaling
     applyResizeMode() {
         if (!UI.rfb) return;
-
-        UI.rfb.scaleViewport = UI.getSetting('resize') === 'scale';
-        UI.rfb.resizeSession = UI.getSetting('resize') === 'remote' || UI.rfb.forcedResolutionX && UI.rfb.forcedResolutionY;
+        let resize_setting = UI.getSetting('resize');
```

Comment:
Minor, but resize_setting is never modified so a const is probably the better choice.